tests/store/test_store_manager.py

```diff
@@ -251,7 +251,7 @@ async def test_addon_version_timestamp(coresys: CoreSys, install_addon_example:
     # If a new version is seen processing repo, reset to utc now
     install_addon_example.data_store["version"] = "1.1.0"
 
-    # Signal the store repositories got updated
-    with patch("supervisor.store.repository.Repository.update", return_value=True):
-        await coresys.store.reload()
-        assert timestamp < install_addon_example.latest_version_timestamp
+    # Touch config file to trigger reload and updated timestamp
+    (coresys.config.path_addons_local / "example" / "config.yaml").touch()
```

Comment:
I don't really like that it depends on modifying fixtures but this seems to be hard to mock :/

Reply:
Hm, yeah I guess I can patch the stat calls instead, feels cleaner.